File: app/src/main/java/com/manimarank/spell4wiki/utils/Print.java
package com.manimarank.spell4wiki.utils;

import android.util.Log;

import com.manimarank.spell4wiki.BuildConfig;

public class Print {

    private final static String TAG = "Spell4Wiki - App";

    public static void log(String message){
        if(BuildConfig.DEBUG){
            Log.d(TAG, message);
        }
    }

    public static void error(String message){
        if(BuildConfig.DEBUG){
            Log.e(TAG, message);
        }
    }
}

File: app/src/main/java/com/manimarank/spell4wiki/utils/Print.kt
package com.manimarank.spell4wiki.utils

import android.util.Log
import com.manimarank.spell4wiki.BuildConfig

/**
 * Utility class for print log data
 */
object Print {
    private const val TAG = "Spell4Wiki - App"
    @JvmStatic
    fun log(message: String?) {
        if (BuildConfig.DEBUG) {
            Log.d(TAG, message ?: "")
        }
    }

    @JvmStatic
    fun error(message: String?) {
        if (BuildConfig.DEBUG) {
            Log.e(TAG, message ?: "")
        }
    }
}